clang/test/SemaCXX/cxx20-using-enum.cpp

```diff
@@ -241,6 +238,13 @@ TPLa<int> a;
 
 } // namespace Thirteen
 
+namespace Fourteen {
+template<typename T>
+int A = T();
+
+using enum A<int>; // expected-error {{A is not an enumerated type}}
+} // namespace Fourteen
```

Comment:
What does `Fourteen` stand for here or is it just a unique name?

Reply:
Yes. If you look at the rest of the file, you'll see tests `One` through `Thirteen`.